archive/config/stargateV1.json

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
why configs `circle`, `hypen`, `maker`, `multichain` were archived as well?

Reply:
Cause we missed those in the past

Reply:
ok, wanted to double check if they were not deleted by mistake